SalesforceCommerceCloud/pwa-kit
# Fix: a11y - Improve accessibility readout for strike through shipping price @W-12627112@

# Description

This PR improves the readout of the shipping options when there is a strikethrough price in the shipping options section on the checkout page. Previously there was no indication that there was a strike through price, so the screen reader would read both the current price and the strikethrough price.

Example: 'Free 1.99'

Now the screenreader explicitly states the original and current price:

Example: 'Originally 1.99, now Free'

![Screenshot 2024-07-11 at 3 50 49 PM](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/88680517/9c1cdcb9-5ff1-4e7a-91ae-e1df21f319ec)




# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

1. `git checkout ju/a11y-shipping-options`
2. start retail react app
3. add products to the cart such that you activate a shipping promotion. On my instance, I have a promotion active such that adding 5 pairs of shoes or more gets you free shipping.
4. Go to the checkout page all the way to shipping options
5. enable the screen reader, for macOS its VoiceOver
6. tab all the way where the screen reader reads out the `Shipping Options` header making sure its in preview mode
7. If you're using voice over, press `command + option + shift + down arrow` and then `option + control + a` in order to get the screen reader to start reading off the text
8. Observe that the screen reader reads the strikethrough price as `Originally ____, now _____`

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
